### ISSUE TITLE
Is it possible to get alignment of the submitted genomes that has the same length as genomes in GTDB ARB file?

### ISSUE THREAD

**Issue by ilnamkang** [Ecogenomics/GTDBTk#35]
Hi,

I'd like to try inserting my genomes into the GTDB ARB tree using several functions available in ARB program. To be able to do this, I need alignment of my genomes that has the same length as genomes in the GTDB ARB file.
But, the alignment length of GTDBTk output file (*.user_msa.fasta) is ~5,000 aa, while the alignment length of GTDB ARB file is ~35,000 aa.
(This difference seems to occur because GTDB and GTDBTk use different masks.)

Are there any method to get the alignment of my genomes that have the same length as genomes in the GTDB ARB file?

Thanks.

**Comment by pchaumeil** [comment]
Hello,

There is currently no options to change the trimming in GTDB-Tk itself.
However one solution would be to replace the masks used by GTDB-Tk by the ones used for GTDB ( available through http://gtdb.ecogenomic.org/downloads )  by pointing the config.py file to those new masks.

Cheers,
Pierre

**Comment by ilnamkang** [comment]
Hi,

Your solution seems to work nicely.

I tested with one of my genomes. When I changed the maks file, GTDBTk stopped during pplacer step. But, I could get the alignment of my genome with the same length as genomes in the ARB file in the output directory. I tried inserting this genome into the ARB tree, and the genome was successfully inserted into a position that is consistent with both GTDBTk classificaiton and my prior knowledge.

Thank you very much for your help.

Ilnam
